tilelang/contrib/nvcc.py

```
@@ -92,6 +92,49 @@ def format_target_code_for_gencode(target_code: object | None) -> str | None:
+        return []
+    try:
+        n = int(raw)
+    except ValueError:
+        return []
+    if n < 0:
+        return []
+
+    try:
+        version = get_cuda_version()  # (major, minor, ...)
+        major, minor = version[0], version[1] if len(version) > 1 else 0
+    except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)